.circleci/config.yml

```diff
@@ -56,6 +56,10 @@ parameters:
     type: string
     # update this as new versions of jaeger become available
     default: "1.33.0"
+  protoc_version:
+    type: string
+    # update this as new versions of jaeger become available
```

Comment:
Is this comment mentioning Jaeger a leftover from duplicating the `jaeger_version` parameter? This is the protoc needed by the [opentelemetry-proto](https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-proto) crate.

Reply:
Yep

Reply:
[f44320d](https://github.com/apollographql/router/pull/1970/commits/f44320d31eaa40cb1a91ad46fe31bf541cbaada0)